Split/Storage/Global/GlobalSecureStorage.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  GlobalSecureStorage.swift
+//  Split
+//
+//  Created by Javier L. Avrudsky on 03/29/2021.
+//  Copyright © 2021 Split. All rights reserved.
+//
+import Foundation
+
+class GlobalSecureStorage: KeyValueStorage {
+
+    static let shared: KeyValueStorage = GlobalSecureStorage()
+
+    func set<T: Encodable>(item: T, for key: SecureItem) {
+        do {
+            let json = try Json.encodeToJson(item)
+            set(item: json, for: key)
+        } catch {
+            Logger.e("Error parsing item \(key.rawValue)")
+        }
+    }
+
+    func get(item: SecureItem) -> String? {
```

Comment:
javi, if this method is not part of KeyValueStorage protocol, maybe it could be private.

Reply:
Thanks Doc! You have a point, but this method could be useful to get a simple string too. That's why I decide to keep it internal instead of private.